Cargo.toml

```diff
@@ -25,8 +32,7 @@ repl = ["async-trait", "rustyline", "dirs-next", "env_logger", "clap", "electrum
 electrum = ["bdk/electrum"]
 esplora = ["bdk/esplora"]
 
-
 [[bin]]
-name = "repl"
-path = "src/repl.rs"
+name = "bdk-cli"
+path = "src/bdkcli.rs"
```

Comment:
nit: I would prefer bdk_cli.rs as the file name (while executable it's right with the hyphen)

Reply:
Sounds good, fixed.